rules/crowdstrike_rules/crowdstrike_dns_request.yml

```diff
@@ -100,4 +101,171 @@ Tests:
+          "111.111.111.111"
+        ],
+        "p_any_domain_names": [
+          "baddomain.com"
+        ],
+        "p_any_trace_ids": [
+          "00000000000000000000000000000001",
+          "00000000000000000000000000000002"
+        ]
+      }
+  -
+    Name: Non-denylisted Domain (FDREvent)
+    ExpectedResult: false
+    Log:
+      {
+        "event_simpleName": "DnsRequest",
+        "name": "DnsRequestMacV1",
+        "aid": "00000000000000000000000000000001",
+        "aip": "111.111.111.111",
+        "cid": "00000000000000000000000000000002",
+        "id": "11111111-0000-1111-0000-111111111111",
+        "event": {
+          "aid": "00000000000000000000000000000001",
+          "event_simpleName": "DnsRequest",
+          "name": "DnsRequestMacV1",
+          "aip": "111.111.111.111",
+          "cid": "00000000000000000000000000000002",
+          "id": "11111111-0000-1111-0000-111111111111",
+          "event_platform": "Mac",
+          "timestamp": "2021-10-01 00:00:00.000Z",
+          "ConfigBuild": "1007.4.0014301.11",
+          "ConfigStateHash": "507116305",
+          "Entitlements": "15",
+          "ContextThreadId": "0",
+          "ContextTimeStamp": "2021-10-08 19:55:04.448Z",
+          "ContextProcessId": "111111111111111111",
+          "EffectiveTransmissionClass": 2,
+          "DomainName": "gooddomain.com",
+          "RequestType": "1",
+        },
+        "event_platform": "Mac",
+        "fdr_event_type": "DnsRequest",
+        "timestamp": "2021-10-01 00:00:00.000Z",
+        "ConfigBuild": "1007.4.0014301.11",
+        "ConfigStateHash": "507116305",
+        "Entitlements": "15",
+        "ContextThreadId": "0",
+        "ContextTimeStamp": "2021-10-08 19:55:04.448Z",
+        "ContextProcessId": "111111111111111111",
+        "EffectiveTransmissionClass": 2,
+        "RequestType": "1",
+        "p_event_time": "2021-10-08 19:55:04.448Z",
+        "p_parse_time": "2021-10-08 20:09:41.933Z",
+        "p_log_type": "Crowdstrike.FDREvent",
+        "p_row_id": "2ed00000000000000000000000000001",
+        "p_source_id": "11111111-1111-1111-1111-111111111111",
+        "p_source_label": "Crowdstrike",
+        "p_any_ip_addresses": [
+          "111.111.111.111"
+        ],
+        "p_any_domain_names": [
+          "gooddomain.com"
+        ],
+        "p_any_trace_ids": [
+          "00000000000000000000000000000001",
+          "00000000000000000000000000000002"
+        ]
+      }
+  -
+    Name: Denylisted Domain (but Non-DNS type) (FDREvent)
+    ExpectedResult: false
+    Log:
+      {
+        "event_simpleName": "Event_DetectionSummaryEvent",
+        "name": "DnsRequestMacV1",
+        "aid": "00000000000000000000000000000001",
+        "aip": "111.111.111.111",
+        "cid": "00000000000000000000000000000002",
+        "id": "11111111-0000-1111-0000-111111111111",
+        "event": {
+          "aid": "00000000000000000000000000000001",
+          "event_simpleName": "DnsRequest",
```

Comment:
Should we maybe change this one too, just to denote that `fdr_event_type` conforms with `event_simpleName`.

Reply:
Thinking again, that there might be a strong argument to not include impossible results in fixtures 😅

Reply:
Not sure I get what you mean here 😄 
Care to elaborate?

Reply:
Resolving as it's outdated.